core/src/main/java/de/danoeh/antennapod/core/syndication/namespace/NSRSS20.java

```diff
@@ -56,7 +56,9 @@ public SyndElement handleElementStart(String localName, HandlerState state,
 			boolean validType = false;
 			boolean validUrl = !TextUtils.isEmpty(url);
 
-			if (type == null) {
+			if(SyndTypeUtils.enclosureTypeValid(type)) {
+				validType = true;
+			} else {
```

Comment:
I think lines 65-67 should be moved into this else clause after `type = ...`
Else, it is a bit confusing why we check the enclose type validity a second time